### PR TITLE
Set target list subscribers via checkbox and select while creating a new form

### PR DESCRIPTION
**Pull request by rosendi**

### CODE REVIEW

File: src/blocks/form/formEditCollectSettings/formEditCollectSettings.js [hunk 1]
```diff
@@ -1,83 +1,114 @@
...
 
-        createNewList: function(event) {
+        _createNewList: function(event) {
             var self = this;
+
             var list = new ListModel({
                 id: null
             });
+
             var modal = new UI.formModal({
```
Comment (Radiansz) [inline src/blocks/form/formEditCollectSettings/formEditCollectSettings.js]: Use require on top instead of UI.

File: src/blocks/select/select.js [hunk 2]
```diff
@@ -94,23 +102,43 @@ define(function(require) {
...
+
+            if (this.resizable) {
+                this.autoresize();
             }
         },
 
-        autoresize: function() {
-            this.$el.find('.select__autoresize option').html(this.$el.find('option:selected').html());
-        }
+        _createNewOption: function() {
```
Comment (Radiansz) [inline src/blocks/select/select.js]: It's rather confusing name. Like create new option in general.
May be change it to something like makeCreationOption?

Comment (rosendi) [inline src/blocks/select/select.js]: @Radiansz  Are we happy with `constructNewOptionInstructions` or just `constructNewOption`?

File: src/blocks/form/formEditCollectSettings/formEditCollectSettings.js (hunk 1 above)
Comment (rosendi) [inline src/blocks/form/formEditCollectSettings/formEditCollectSettings.js]: Yeah :-) It's done.

File: src/blocks/select/select.js (hunk 2 above)
Comment (Radiansz) [inline src/blocks/select/select.js]: I think second'll be fine

Comment (rosendi) [inline src/blocks/select/select.js]: Done.

**Comment by Radiansz** [comment]
LGTM

**Pull request merged**